index.js

```diff
@@ -83,15 +85,25 @@ const MultiselectPrompt = {
 		this.render()
 	}
 	, right: function () {
-		this.value[this.cursor].selected = true
-		this.render()
+		if (this.value.filter (e => e.selected).length < this.maxChoices) {
+			this.value[this.cursor].selected = true
+			this.render()
+		}
+		else
+			return this.bell()
 	}
 
 	, _: function (c) { // on space key
 		if (c !== ' ') return this.bell()
 		const v = this.value[this.cursor]
-		v.selected = !v.selected
-		this.render()
+
+		if (v.selected) v.selected = false
```

Comment:
minor stylistic grumble:

I'd like the `_` function to fail fast. So please put the `>= this.maxChoices` check in the beginning and `return`.